destalinator.py

```diff
@@ -125,7 +125,7 @@ def stale(self, channel_name, days):
             for x
             in messages
             if x.get("user") not in self.config.ignore_users
-            and x.get("text")
+            and (x.get("text") or x.get('attachments'))
```

Comment:
This allows messages from many integrations to count. For instance, PagerDuty only uses `attachments`.